.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules/
 coverage/
+*.lock
+.idea/
```

Comment:
Sorry, not interested in IDE specific ignores.

Reply:
I believe that it makes it simpler when you make commit to make sure that you aren't adding useless metadata or IDE specific that can cause issues.